scargo/scargo_src/sc_docker.py

```diff
@@ -52,15 +54,17 @@ def scargo_docker_run(
     docker_path = _get_docker_path()
     project_config_name = _get_project_config().name
 
-    cmd = " ".join(
-        filter(
-            None,
-            ["docker-compose run", *docker_opts, f"{project_config_name}_dev", command],
-        )
-    )
+    cmd = [
+        "docker-compose",
+        "run",
+        *docker_opts,
```

Comment:
What happens here if docker_opts is None?

Reply:
To answer your question: it would raise a `TypeError`. Same for an `int` or anything else that isn't iterable.

Reply:
Maybe this was for something else I was thinking. Sequence should be right